x/ibc/03-connection/keeper/keeper.go

```diff
@@ -128,13 +128,6 @@ func (k Keeper) VerifyMembership(
 	pathStr string,
 	value []byte,
 ) bool {
-	// FIXME: commented out for demo
```

Comment:
why was this comment deleted?

Reply:
Was a leftover comment that we don’t need anymore